plugins/org.wso2.developerstudio.eclipse.gmf.esb.edit/src-gen/org/wso2/developerstudio/eclipse/gmf/esb/parts/forms/SequencePropertiesEditionPartForm.java

```diff
@@ -1287,32 +1290,28 @@ protected Composite createDynamicReferenceKey(FormToolkit widgetFactory, final C
 		widgetFactory.paintBordersFor(parent);
 		GridData valueData = new GridData(GridData.FILL_HORIZONTAL);
 		dynamicReferenceKeyText.setLayoutData(valueData);
-		dynamicReferenceKeyText.addFocusListener(new FocusAdapter() {
-			/**
-			 * @see org.eclipse.swt.events.FocusAdapter#focusLost(org.eclipse.swt.events.FocusEvent)
-			 * 
-			 */
-			@Override
-			@SuppressWarnings("synthetic-access")
-			public void focusLost(FocusEvent e) {
-			}
 
-			/**
-			 * @see org.eclipse.swt.events.FocusAdapter#focusGained(org.eclipse.swt.events.FocusEvent)
-			 */
-			@Override
-			public void focusGained(FocusEvent e) {
-				EEFNameSpacedPropertyEditorDialog nspd = new EEFNameSpacedPropertyEditorDialog(parent.getShell(),
-						SWT.NULL, dynamicReferenceKey);
-				// valueExpression.setPropertyValue(valueExpressionText.getText());
-				nspd.open();
-				dynamicReferenceKeyText.setText(dynamicReferenceKey.getPropertyValue());
-				propertiesEditionComponent.firePropertiesChanged(new PropertiesEditionEvent(
-						SequencePropertiesEditionPartForm.this,
-						EsbViewsRepository.Sequence.Properties.dynamicReferenceKey,
-						PropertiesEditionEvent.COMMIT, PropertiesEditionEvent.SET, null, getdDynamicReferenceKey()));
-			}
-		});
+		dynamicReferenceKeyText.addMouseListener(new MouseAdapter() {
```

Comment:
Fix formatting issues.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/804